src/note2.js

```diff
@@ -0,0 +1,30 @@
+const wordsUnique = [
+    'crab',
+    'poison',
+    'contagious',
+    'simple',
+    'bring',
+    'sharp',
+    'playground',
+    'poison',
+    'communion',
+    'simple',
+    'bring'
+  ];
+  
+function uniquifyArray(vocabulary) {
+    newArray=vocabulary;
```

Comment:
To create a new array, you need to use
`const newArray = []` or
`const newArray = new Array(<amount of elements>)`